packages/tx/src/7702/tx.ts

```diff
@@ -119,8 +119,8 @@ export class EOACodeEIP7702Transaction extends BaseTransaction<TransactionType.E
   /**
    * The amount of gas paid for the data in this tx
    */
-  getDataFee(): bigint {
-    return EIP7702.getDataFee(this)
+  getDataGas(): bigint {
```

Comment:
Can someone remind me why we wrap this helper function instead of just having the code here?

Reply:
To avoid redundancy, since we have this code in various tx types.